Formula/p/porter.rb

```diff
@@ -0,0 +1,27 @@
+class Porter < Formula
+  desc "App artifacts, tools, configs, and logic packaged as distributable installer"
+  homepage "https://porter.sh"
+  url "https://github.com/SpiffyEight77/porter.git",
+    tag:      "v1.0.17",
+    revision: "07959e4fb02296895fc3b115ecd61f93b984a7dc"
```

Comment:
```suggestion
  homepage "https://porter.sh/"
  url "https://github.com/getporter/porter/archive/refs/tags/v1.0.17.tar.gz"
  sha256 "5262251aba930331b0e3b83cb971f6d138611b37b81af621606b6578739b5e9a"
```

Reply:
maybe next release, relates to https://github.com/getporter/porter/issues/3064

Reply:
> maybe next release, relates to [getporter/porter#3064](https://github.com/getporter/porter/issues/3064)

@chenrui333 
This issue has been resolved, and I am discussing with the maintainer the possibility of re-releasing v1.0.17 with the latest dependency, relates to https://github.com/getporter/porter/issues/3045

Can I specify a commit revision for the build formula? The process uses git commands, but the tar.gz file doesn't include a .git directory. 🙇🏻‍♂️

```
  homepage "https://porter.sh"
  url "https://github.com/getporter/porter.git",
  tag:      "v1.0.17",
  revision: "605e399737c5c89f0ce692c6aa3d9e6211250c95"
```